.github/workflows/build.yml

```diff
@@ -51,6 +51,9 @@ jobs:
     name: Test Suite
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Install dependencies
+        run: sudo apt-get install libxcb-xfixes0-dev
```

Comment:
We should add a comment to the README that `libxcb` is required to build on Linux. It should be already present on X11 systems but might be missing for Wayland users (maybe we should make `x11` a feature that's on by default so people can opt out).

Reply:
The option makes sense for that reason. Alacritty does that as well.